Cargo.toml

```
@@ -8,16 +8,20 @@ license       = "Apache-2.0"
 repository    = "https://github.com/CosmWasm/cw-nfts"
 homepage      = "https://cosmwasm.com"
 documentation = "https://docs.cosmwasm.com"
+rust-version = "1.65"
```

Align with other lines

```suggestion
rust-version  = "1.65"
```